### DISCUSSION TITLE
유저 추가 요청

### DISCUSSION THREAD

**Discussion by devkunst** [Pseudo-Lab/9th-builder#34] [Issue and Q&A] [answered]
mumunuu (jae_won@icloud.com)
jeongminia (ajm101683@gmail.com)
leh60245 (kkh60245@gmail.com)
shinhyunji36 (lej70445@gmail.com)
thsamajiki (ant1102@naver.com)
ddinggul (litterrariuschwy@gmail.com)

계정 추가 부탁드립니다. 
https://github.com/Pseudo-Lab/pseudolab-study-ai-application-with-llm/pull/1
PR 에 다음 인원이 추가되지 않습니다.

**Comment by jasonjo97** [comment, marked as answer]
안녕하세요 @seriousran 님, 가짜연구소 맴버에 해당 디스코드 아이디 (깃헙 아이디) 추가 부탁드립니다! 

류지혜 - yihaeroo (Domwis-IR)
김기용 - gimgiyong5910 (kimk-ki)
유경민 - pixeldosa (yukyeongmin)
신찬영 - eren_ai (ChanYeongShin)
이중원 - oneoftwo942 (oneoftwo)

감사합니다 :)

**Comment by devkunst** [comment]
진행이 되었으므로 해당 디스커션을 닫겠습니다. 감사합니다.

**Comment by devkunst** [comment]
-
